content/doc/addons/keycloak.md

```diff
@@ -0,0 +1,181 @@
+
+To fit the most common needs, the Keycloak add-on comes with batteries included: sensible defaults, easy migration with realms import/export, Keycloak Metrics, default theme, etc.
+
+We have partnered with [*Please Open-it*](https://please-open.it/) to develop this add-on and to offer the most efficient and straightforward Keycloak service. They are experts for many years and are a go-to reference for advanced deployments and consulting.
+
+## Technical preview? What does that mean?
+
+We are proud to release great products, fit for the industry, and the easiest to use. To do that, we need your [feedback](https://www.clever-cloud.com/fr/contact-sales/). We also want to be sure we have worked in a good direction. Hence, we have to fire-test the product. That is why the Keycloak add-on is first **offered as a Tech Preview**, before releasing it for production use.
+
+During the Tech Preview period:
+- Features consistencies are battle-tested and may evolve; 
+- UX and interface are not final and may also evolve;
+- Bugs and potential issues feedback are deeply sought;
+- Features and pricing may evolve;
+- 0 € service & maintenance fees.
+
+## Key features
+Keycloak on Clever Cloud allows you to effortlessly set up a tailored authentication and access management solution, that can be adjusted to your needs and workloads. It offers a wide panel of services such as:
+- Secure Identity Management;
+- Single Sign-On (SSO);
+- Centralized Administration;
+- Customizable Authentication (OAuth 2.0, LDAP, SAML, OpenID Connect, and more...);
+- Easily load plugins and themes;
+- Multi Realms;
+- Monitoring and Logging;
+- Export and Import;
+- Support and Maintenance.
+
+## Need expert advice?
+The Clever Cloud Keycloak add-on is designed to meet the most common needs. However, realizing how central and critical authentication needs are for companies, we decided to tackle the issue and recommend expert advice for the most complex and loaded systems.
+
+[*Please Open-it*](https://please-open.it/) are not only the Keycloak experts who helped us build a great add-on, but they also are recognized experts in authentication and access management, with years of experience. They are here to help architect and configure complex authentication projects. [Contact them directly for more information](mailto:sales@clever-cloud.com)__.
+
+## How it works?
+When you subscribe to the Keycloak add-on, we automatically deploy a [Java](https://developers.clever-cloud.com/doc/applications/java/java-jar/) instance with Keycloak pre-loaded and configured. We also deploy a [PostgreSQL](https://developers.clever-cloud.com/doc/addons/postgresql/) database and an [FSBucket](https://developers.clever-cloud.com/doc/addons/fs-bucket/) used for themes, plugins, and import/export storage needs. 
```

Comment:
Use bucket lists instead of paragraphs (grabs the attention better and simplifies the reading)

```suggestion
When you create the Keycloak add-on, Clever Cloud automatically deploys:

-  A [Java](https://developers.clever-cloud.com/doc/applications/java/java-jar/) instance with Keycloak pre-loaded and configured
- A [PostgreSQL](https://developers.clever-cloud.com/doc/addons/postgresql/) database
- An [FSBucket](https://developers.clever-cloud.com/doc/addons/fs-bucket/) used for themes, plugins, and import/export storage needs. 
```